src/IECoreHoudini/GEO_CobIOTranslator.cpp

```diff
@@ -163,3 +164,27 @@ GA_Detail::IOStatus GEO_CobIOTranslator::fileSaveToFile( const GEO_Detail *geo,
 	
 	return fileSaveToFile( geo, fileName );
 }
+
+bool GEO_CobIOTranslator::fileStat( const char *fileName, GA_Stat &stat, uint level )
+{
+	try
+	{
+		ReaderPtr reader = Reader::create( fileName );
+		if ( !reader )
+		{
+			return false;
+		}
+		
+		CompoundObjectPtr header = reader->readHeader();
+
+		UT_BoundingBox bbox = convert<UT_BoundingBox>( reader->readHeader()->member<const Box3fData>( "bound", true )->readable() );
```

Comment:
maybe I misunderstood something from cortex API. My intention was to throw an exception if `bound` is missing that means `setBounds` won't be called and `false` will be returned. 
```C++
template<typename T >
T* IECore::CompoundData::member(const InternedString &name, bool throwExceptions = false)	
```

Do you want me to make it more explicit ?